src/main/generated/options/soot/AntTask.java

```diff
@@ -223,6 +223,10 @@ public Path createprocess_dir() {
             return process_dir.createPath();
         }
   
+        public void setderive_java_version(boolean arg) {
```

Comment:
@pavanupb Here is the stuff? What are you missing?

Please just make sure that you do not remove anything I added in this mr with your merge with develop. All files in the `src/main/generated` folder will be generated when calling `mvn verify`

Reply:
@mbenz89 I executed the maven command mvn clean verify but still the files are not getting generated. Am I missing something?

Reply:
Are you sure you did not overwrite my changes with the ones from develop?

Reply:
Yes I took your changes as well as the new changes from develop